lib/components/dropdown-item.vue

```diff
@@ -7,6 +7,7 @@
        tabindex="-1"
        role="menuitem"
        @click="click"
+       @click.native="click"
```

Comment:
Nice idea :) Just as a question won't having both makes duplicate events? Why we kept both?